doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-creating-a-service-for-provisioning-vm.adoc

```diff
@@ -0,0 +1,169 @@
+[appendix]
+
+[[example-creating-a-service]]
+== Example: Creating a Service for Provisioning a Virtual Machine through Red Hat Virtualization (RHV)
+
+{product-title} provides the ability to define services, which can include one or more virtual machines or instances, and can be deployed across hybrid environments. Services can be made available to users through the service user interface (SUI) that allows users to order predefined IT services without IT operations getting involved.
+
+The following three items are required to make a service available to users from the {product-title_short} service user interface:
+
+. *Service Dialog*: A service dialog is where you allow users to configure virtual machine or instance options. A service dialog determines which options users get to change. You can choose the options presented to the user. You could just give them the option to set the service name, or you could have them change all of the provisioning dialog options. 
```

Comment:
I might edit down the sentences starting with "You can choose the options presented to the user" to consume the idea expressed in the following sentence. Maybe something like, "You can choose the options presented to the user, from setting the service name to changing all provisioning dialog options."